generate_upstream_list.py

```diff
@@ -9,27 +9,23 @@
 
 def get_commit_log(git_repo, viewvc_url, author_in_commit_body=False):
     repo = Repo(git_repo)
-    log = repo.commits(config.EMAIL_GREP, search_body=author_in_commit_body)
+    email_grep = '(Contributed|Patch) (from|by).*@opera.com'
```

Comment:
It would be nice not to hardcode, since I saw other people using this as their upstream (for similar pages) :) That's why the config file is there.